### DISCUSSION TITLE
You are using multiple accounts (web wallet)

### DISCUSSION THREAD

**Discussion by ASX320** [revoxhere/duino-coin#1727] [Q&A]
Hi, I've met an issue with DuinoCoin wallet on the web (https://wallet.duinocoin.com). I can't perform any kind of operations with coins. I've made a screenshot:

![Screenshot 2023-12-31 at 5 03 04 PM](https://github.com/revoxhere/duino-coin/assets/96183740/b7072877-1053-4278-bd8c-5cd63059518e)

The listed accounts **do not belong to me**, my is k050506koch. If someone knows how to resolve this issue, help please

**Comment by TiberiusSky** [comment]
"Wolliges_Schaf" have the same problems. He describes it on discord. Very strange issue!
I have also some trouble with multiple accounts. Can't do anything with my coins :(
One reason could be, we in Germany getting rotating public IP's from our providers. It can be possible that different Duino-Users get the same IP over a large timeperiod.   
I wrote many times to support, but without feedback.

**Reply by ArakelTheDragon** [reply]
Hi!

I also tried to solve this problem as I really need to trade my coins through metamask and pancakeswap, I have 2500 coins already, which are difficult to mine, the send and wrap function, both don't work. Is there any way to solve this? The only thing I could think of is really make another account and start mining again.

**Comment by makerpro299** [comment]
I have the same issue and can't do anything with my account, please fix this. I am not using a VPN, Private Relay or Proxy.

**Comment by Sule270774** [comment]
how to delete multiple accounts because they are not for transactions. The account I currently maintain is Sule270774
Thanks..

**Comment by Gab729** [comment]
We are in 2025 and the problem still there, i just discover duino and cant verify my account because some account i nevers herd off. And verification team denied for same reason... No awnser from support
![Screenshot_20250130-021318](https://github.com/user-attachments/assets/e76cfa53-df45-44ea-b663-d32a05e19a2b)

Hope aomeone ear about a solution

**Comment by extfatkur** [comment]
the problem still happen. 
"you're using multiple account...."

**Comment by pspcfactory** [comment]
<img width="1180" height="820" alt="IMG_0159" src="https://github.com/user-attachments/assets/e8c07c77-68d2-4c26-81a5-b9232a78ba0e" />
Yeah I to have the same issue Multiple Account using . How to solve this problem I can’t send the coins from my seccond account while first one is working but all my coins are in my 2nd account . Mr. Revox Please heed everyones issue and solve this

**Reply by linastorvaldz** [reply]
@revoxhere
